1st pass of rebrand to Open Supply Hub

maurizi opened the pull request:
## Overview

Makes a few small tweaks to the site branding / logo for the beta period

Connects #1879 

## Demo

![localhost_6543_](https://user-images.githubusercontent.com/4432106/173126577-7f28a148-2cdb-44ef-b23a-d3ed296a0ded.png)
![localhost_6543_ (1)](https://user-images.githubusercontent.com/4432106/173126652-c851058a-ee01-420b-8d67-958099754d2f.png)


## Notes

I just made some guesses on how to style the login/logout/translation buttons in the navbar.

My assumption from "Make map greyscale" was that it applied to the facilities layer which had 4 blue colors (and not the basemap).
My first pass was to directly use the grayscale versions of the existing colors, but these were very hard to distinguish, so I settled on the current colors after some simple trial & error, but it's definitely quite basic.

I _could_ have made the primary button color on the entire site green to fulfill the requirements of "Changing the search button color", but based on the limited scope of the issue I left it at just the search button.

## Testing Instructions

* `scripts/server`
* Observe that the main map screen now has a green header with a grayscale map layer, and that the search button is also green

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- ~[ ] If this PR applies to both OAR and OGR a companion PR has been created~

jwalgran reviewed (commented):
There is some missing detail in the issue description related to "greyscale map"

We already have support for a greyscale styling of the basemap inside the VectorTileFacilitiesMap component 

https://github.com/open-apparel-registry/open-apparel-registry/blob/3637f8baac2f6fa22de21be04e3fa5f3133eb532/src/app/src/components/VectorTileFacilitiesMap.jsx#L133

This was added so that embedded map users who use an iframe have a more neutral option that may work better with their surrounding brand colors.

We should be able to switch this option on for the main map and then restyle the dots to a green gradient rather than a blue one.

maurizi commented:
> There is some missing detail in the issue description related to "greyscale map"

Oh this is much simpler. Rebased to remove the old commit, and switched to this in https://github.com/open-apparel-registry/open-apparel-registry/pull/1900/commits/87992bc5568d6fca131dbc2a6d10069ac5805484

maurizi commented:
![localhost_6543_ (2)](https://user-images.githubusercontent.com/4432106/173143207-0c4096fa-c77d-462b-8795-e894d0da1813.png)

jwalgran commented on src/app/src/components/Navbar/AuthMenu.jsx:
```diff
@@ -24,13 +24,13 @@ import { convertFeatureFlagsObjectToListOfActiveFlags } from '../../util/util';
 
 const styles = {
     profileButton: {
-        color: 'white',
```
We have an existing `COLORS.js`  so where it makes sense we could introduce a constant and make future tweaking easier.

maurizi replied:
I think I actually looked for `COLORS.js` on my own once and missed it, since it's currently `COLOURS.js` 🇬🇧

jwalgran reviewed (approved):
Confirmed!
